src/indieweb_utils.py

```diff
@@ -345,28 +345,37 @@ def discover_web_page_feeds(url):
 
     # check for presence of mf2 hfeed
     h_feed = soup.find_all(class_="h-feed")
+    page_title = soup.find("title")
+
+    page_domain = url.split("/")[2]
+
+    valid_mime_types = [
```

Comment:
This is probably better define _outside_ of this function, but it can wait until we start dividing this file into a module. Also, since we want just unique values, perhaps this is better off as a set, rather than a list (assuming we don't care about order)?

Reply:
I have changed the valid_mime_types value to a set.